data/json/items/armor/ammo_pouch.json

```diff
@@ -675,6 +859,22 @@
     "proportional": { "weight": 0.75, "volume": 0.75, "price": 0.5 },
     "extend": { "flags": [ "UNDERSIZE", "PREFIX_XS" ] }
   },
+  {
+    "id": "xl_tacvest_sling",
+    "type": "ARMOR",
+    "copy-from": "tacvest_sling",
+    "name": { "str": "tac vest with sling", "str_pl": "tac vests with slings" },
+    "proportional": { "weight": 1.25, "volume": 1.13, "price": 1.25 },
+    "extend": { "flags": [ "OVERSIZE", "PREFIX_XL" ] }
+  },
+  {
+    "id": "xs_tacvest_sling",
+    "type": "ARMOR",
+    "copy-from": "tacvest_sling",
+    "name": { "str": "tac vest with sling", "str_pl": "tac vests with slings" },
```

Comment:
```suggestion
    "name": { "str": "XS tac vest with sling", "str_pl": "XS tac vests with slings" },
```

Reply:
Unsure how, but in my testing these actually had the correct name in the item spawn window? 
![image](https://github.com/user-attachments/assets/aa69763b-e33f-435b-9eae-7034a4069874)
Applied changes regardless